src/ng-stomp.js

```diff
@@ -28,14 +28,20 @@ angular
         var dfd = $q.defer()
 
         this.sock = new SockJS(endpoint)
+        this.sock.onclose = function () {
+          if (angular.isFunction(errorCallback)) {
+            errorCallback(new Error('Connection broken'))
```

Comment:
A single minor concern: Is `onclose` only triggered for "broken connections"? Or does this conflict with regular disconnects (e.g. via `stomp.disconnect()`)?